src/current/v23.2/cockroach-debug-merge-logs.md

```diff
@@ -39,44 +40,48 @@ Generate a debug zip file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
+cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
 ~~~
 
 Unzip the file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ unzip ./cockroach-data/logs/debug.zip
+unzip ./cockroach-data/logs/debug.zip
 ~~~
 
 Merge the logs in the debug folder:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug merge-logs debug/nodes/*/logs/*
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1
```

Comment:
I notice these examples use the `crdb-v1` format but https://www.cockroachlabs.com/docs/stable/log-formats calls it a "legacy format".  Are we still encouraging the use of `crdb-v1`?  What would happen if these commands used `crdb-v2`?  Is the upshot here that I as a user need to know/care which logging version was in use if I'm going to run `merge-logs` ?

Reply:
Ah ok after reading this again, I see that this example is looking at the v1 logs created by `debug zip`

I'm kinda bummed that as a user I need to care about these different formats :-/ (but that is beyond the scope of this docs PR obv)

Reply:
OK I'm less bummed now that I know I'm the one configuring the formats via https://www.cockroachlabs.com/docs/v23.2/configure-logs#yaml-payload 

Sorry for the noise on my "learning journey" in this PR :-)